neo/rawio/tdtrawio.py

```diff
@@ -248,14 +249,21 @@ def _parse_header(self):
                         sev_filename = (path / sev_stem).with_suffix('.sev')
                     else:
                         # for single block datasets the exact name of sev files in not known
-                        sev_regex = f".*_ch{chan_id}.sev"
+                        sev_regex = f"*_Ch{chan_id}.sev"
                         sev_filename = list(self.dirname.parent.glob(str(sev_regex)))
+                        # in case multiple sev files are found, try to find the one for current stream
+                        if len(sev_filename) > 1:
```

Comment:
I think this can be simplified a bit via
```suggestion
                        if sev_filename:
```

Reply:
This will not work with the test data, as when len(sev_filename) is exactly one, the format of the file is not necessarily f'*_{store}_Ch{chan_id}.sev'. I think the initial suggestion is more appropriate